spec/flipper/adapters/failover_spec.rb

```diff
@@ -84,21 +84,19 @@
   context 'when primary is instrumented and fails' do
     before do
       allow(memory_adapter).to receive(:get).and_raise(Net::ReadTimeout)
+      Flipper.instrumenter = instrumenter
```

Comment:
This is ok because it's actually updating config and config is reset for every spec, right? Just did a double take because global changes without resetting always make me shiver. :)